### ISSUE TITLE
EPUB cover image and author information not displaying as expected

### ISSUE THREAD

**Issue by geealbers** [thegetty/quire#715]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

1. Start a new Quire project
2. Run `quire build` and then `quire epub` to create the EPUB file
3. Run the EPUB through epubcheck with `java -jar path/to/epubcheck.jar patt/to/epubjs.epub`

### Actual behavior

We're including a static cover image and creator information in the `_epub/manifest.json` file, but neither are showing up where expected. For instance when opening the file in Apple Books a thumbnail of the cover should appear, but it's only a Apple's default background, the book title, and "Unknown Author". The snippet of the manifest is below. I believe the cover file path is correct, not sure if the structure/naming of the JSON itself doesn't match epub.js expectations.

```json
{
  "@context": [
    "https://schema.org",
    "https://www.w3.org/ns/pub-context"
  ],
  "conformsTo": "https://www.w3.org/TR/pub-manifest/",
  "contributors": [],
  "cover": "_assets/images/static-cover.jpg",
  "creators": [
    {
      "name": "Judith Keller",
      "role": "aut",
      "file-as": "Keller, Judith"
    }
  ],
```

<img width="433" alt="Screen Shot 2023-04-21 at 3 13 29 PM" src="https://user-images.githubusercontent.com/7796401/233742364-7fc8d0a1-291e-4478-902d-ab40b1eae973.png">

### Expected behavior

I'd expect to see the cover image in in the e-book reader interface in various places, and the author to be listed.

### Version numbers

quire-cli 1.0.0-rc.6
quire-11ty 1.0.0-rc.8
node 18.12.1

**Comment by fchasen** [comment]
This should be in `resources` as:

```
"resources": [
        {
            "url": "_assets/images/static-cover.jpg",
            "rel": "cover-image"
        }
]
```

but I think this is a common enough use-case when creating Epubs that I should update the CLI to look for it.

**Comment by fchasen** [comment]
Added the cover parsing for the next epub.js release, and fixed the `key.name` issue that was preventing the creators from working in the CLI.

**Comment by geealbers** [comment]
@fchasen The creator is still not being added to the package.opf file correctly:

```html
<dc:creator>[object Object]</dc:creator>
```
I assume it has something to do with the way the info is structured in the manifest.json we're feeding in? Can you advise? This is what we have in manifest.json now:

```
  "creators": [
    {
      "name": "Judith Keller",
      "role": "aut",
      "file-as": "Keller, Judith"
    }
  ],
```

**Comment by fchasen** [comment]
Thanks, for that snippet - epubjs-cli was looking for the wrong key.
Just released `0.1.4` with a fix for that.

**Comment by geealbers** [comment]
Awesome. Thanks for the quick response!
